developer/tutorial/plugin-layouts-3.md

```diff
@@ -19,7 +19,7 @@ import classnames from "classnames";
 import Blaze from "meteor/gadicc:blaze-react-component";
 import { Template } from "meteor/templating";
 
-import { getComponent as assertComponent, registerComponent } from "/imports/plugins/core/components/lib";
+import { getComponent as assertComponent, registerComponent } from "@reactioncommerce/reaction-components";
```

Comment:
This would require to have reaction-components aliased in plugin's package.json, which is not done automatically (maybe it should?) when creating the plugin via `reaction plugins create --name <plugin_name>`. I'd leave this as is for the moment and create an issue for reaction-cli instead.

Reply:
Also these code changes should be in sync with https://github.com/reactioncommerce/reaction-example-plugin/blob/master/client/templates/layouts/core.js